alerting/notifier/channels/kafka.go

```diff
@@ -48,12 +54,36 @@ type KafkaNotifier struct {
 }
 
 type kafkaSettings struct {
-	Endpoint    string `json:"kafkaRestProxy,omitempty" yaml:"kafkaRestProxy,omitempty"`
-	Topic       string `json:"kafkaTopic,omitempty" yaml:"kafkaTopic,omitempty"`
-	Description string `json:"description,omitempty" yaml:"description,omitempty"`
-	Details     string `json:"details,omitempty" yaml:"details,omitempty"`
+	Endpoint       string `json:"kafkaRestProxy,omitempty" yaml:"kafkaRestProxy,omitempty"`
+	Topic          string `json:"kafkaTopic,omitempty" yaml:"kafkaTopic,omitempty"`
+	Description    string `json:"description,omitempty" yaml:"description,omitempty"`
+	Details        string `json:"details,omitempty" yaml:"details,omitempty"`
+	BasicAuth      bool   `json:"basicAuth,omitempty" yaml:"basicAuth,omitempty"`
+	BasicAuthUser  string `json:"basicAuthUser,omitempty" yaml:"user,omitempty"`
+	PasswordSource string `json:"passwordSource,omitempty" yaml:"passwordSource,omitempty"`
+	PassFilePath   string `json:"passwordFilePath,omitempty" yaml:"passwordFilePath,omitempty"`
+	AuthPass       string `json:"password,omitempty" yaml:"password,omitempty"`
```

Comment:
Let's rename it to Password to match the name of the json field.

Reply:
Agreed, I'll make the change!

Reply:
Done!